robottelo/ui/contentviews.py

```diff
@@ -55,18 +55,13 @@ def create(self, name, label=None, description=None, is_composite=False):
         self.wait_for_ajax()
         self.click(common_locators['create'])
 
-    def delete(self, name, really=True):
+    def delete(self, name):
```

Comment:
* Is content view only entity which has this kind of a form confirmation?
* self.delete_entity() doesn't seem to handle this situation.  I believe it is better to leave this code as it is.